supplementary_style_guide/glossary_terms_conventions/general_conventions/v.adoc

```diff
@@ -202,6 +202,17 @@ Although the _IBM Style Guide_ recommends using "diskette drive" instead of "flo
 
 *See also*:
 
+[discrete]
+[[volatile-storage]]
+==== image:images/yes.png[yes] Volatile storage (noun)
```

Comment:
Can "Volatile" be lowered to "volatile" (since I assume it doesn't have to start with a capital V if it's in the middle of a sentence)?